src/routes/assistants/+page.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+	import { goto } from "$app/navigation";
+	import { base } from "$app/paths";
+	import { page } from "$app/stores";
+	import type { PageData } from "./$types";
+	export let data: PageData;
+	let selectedModel = $page.url.searchParams.get("modelId") ?? "";
```

Comment:
```suggestion
	import type { PageData } from "./$types";

	export let data: PageData;

	let selectedModel = $page.url.searchParams.get("modelId") ?? "";
```

nit: up to you to accept this suggestion or not